src/jasper_helpers/links.cr

```diff
@@ -15,15 +15,27 @@ module JasperHelpers::Links
     link_to(yield, url, **options)
   end
 
-  def button_to(body : String, url : String, method = :post, **options : Object)
-    form(action: url, class: "button", method: method) do
-      content(:button, body, options)
+  def button_to(body : String, url : String, method : Symbol = :post)
```

Comment:
Why did you remove options from here?

Reply:
That got moved down kinda. I reorder the methods to show the progression of the overloading. So it shows as 

Basic usage 
Basic usage with yield 
Basic usage with yield and options